### DISCUSSION TITLE
Possible to connect to gui from another computer on the same network?

### DISCUSSION THREAD

**Discussion by caseyroot97** [koush/scrypted#216] [Q&A] [answered]
Hi, basically just what the title says. I've been using homebridge and have been able to connect to their gui from another computer on the same network by entering the ip of the raspberry pi it's running on plus the port it's using as the url. When I try to do that with scryted however, my browser says the server unexpected dropped the connection. I'm definitely a novice when it comes to networking so I'm not sure whats goin on here or if its not possible with scrypted for some reason.

**Comment by koush** [comment, marked as answer]
yes, it's https, which is why it may be failing to connect. use https://ip:10443/

**Reply by caseyroot97** [reply]
Wow I thought I tried that but it worked, thanks!
